site/content/en/guides/consumer/compositepkgs/_index.md

```diff
@@ -0,0 +1,171 @@
+---
+title: "Get and apply a remote composite package"
+linkTitle: "Composite Package"
+weight: 7
+type: docs
+description: >
+  Fetch a composite package from a remote git repository and apply its contents to
+  a cluster
+---
+
+{{% pageinfo color="warning" %}}
+
+#### Notice: Composite packages feature support is in alpha phase
+
+{{% /pageinfo %}}
+
+_A kpt package is a directory of resource configs with valid Kptfile in it.
+A composite package is a kpt package with 1 or more subpackages in it._
+
+## Steps
```

Comment:
There are other steps here not captured in this list such as apply. Per the other command, guide user through each step to accomplish an objective.

Reply:
I added more information and a flow to achieve a task. PTAL.